src/core/main.c

```diff
@@ -217,6 +226,8 @@ int main(int argc, char *argv[]) {
 	}
 
 	// TODO: cleanup
+	finish_config(&config);
+	free((char *)configfile);
```

Comment:
Something that we need to free can't be `const`

Reply:
Hope i got the const and not const declaration right this time.